tool/caffe_model/cifar10_full_train_test.prototxt

```diff
@@ -0,0 +1,220 @@
+name: "CIFAR10_full"
```

Comment:
move this into examples/cifar10/caffe
and add another file, e.g. caffe.py for training using the prototxt files.